_posts/2018-11-17-elections-2019.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Announcing the third Vox Pupuli Elections
+date: 2018-11-17
+github_username: binford2k
+twitter_username: binford2k
+---
+
+Vox Pupuli has opened elections for the 2019 calendar year Project Management Committee.
+The elections were announced at Puppet's [Contributor Summit Online][cso].
+This blogpost will give you the context and how to become a candidate.
+
+> Vox Pupuli is a collective of Puppet module, tooling and documentation authors
+> all working together to ensure continued development on the code we maintain.
+
+(Source: [https://voxpupuli.org/](https://voxpupuli.org/))
+
+The Vox Pupuli project is now more than three years old. It represents many
```

Comment:
I'm not actually sure what date is considered to be the genesis date of VP, so someone please correct this :)

Reply:
I think nobody knows :D